hl7_2_ada/hl7/hl7_2_ada_hl7_include.xsl

```diff
@@ -418,7 +424,10 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     </xsl:call-template>
                 </xsl:when>
                 <xsl:otherwise>
-                    <xsl:message terminate="yes">Cannot determine the datatype based on @xsi:type, or value not supported: <xsl:value-of select="$xsiType"/>. Calculated QName <xsl:value-of select="$xsiTypeURIName"/></xsl:message>
+                    <xsl:call-template name="util:logMessage">
+                        <xsl:with-param name="level" select="$logWARN"/>
```

Comment:
This should be FATAL based on `terminate="yes"` in old situation.

Reply:
Agree!

Reply:
Agree